examples/tutorials/yandex_faq.ipynb

```diff
@@ -363,13 +344,20 @@
     "\n",
     "[DeepPavlov blog](https://medium.com/deeppavlov)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "dp_dev",
    "language": "python",
-   "name": "python3"
+   "name": "dp_dev"
```

Comment:
Will this work in google colab as python3 by default? Or will it be launched as a python2 instance?

Reply:
I'm using my own env to test my local version of deeppavlov. It has python3 on board.